SPECS/kernel/kernel-uki.spec

```diff
@@ -13,12 +13,12 @@
 # We need to temporarily turn it off by disabling xsaves until the problem
 # is fixed on Azure. Since shadow stack depends on xsaves, disabling xsaves
 # ensures the feature bit for shadow stack is also turned off.
-%define cmdline console=ttyS0 noxsaves
+%define cmdline console=ttyS0
```

Comment:
Is this really all we need for kernel-uki commandline?

Reply:
We might want to add more to make it on par with the regular marketplace image, but this is all we need to make the image work for now

Reply:
changes lgtm